plugins/binanceExchange_ws.go

```diff
@@ -215,12 +227,14 @@ func (beWs *binanceExchangeWs) GetTickerPrice(pairs []model.TradingPair) (map[mo
 			}
 
 			//Store stream
-			beWs.mtx.Lock()
-			beWs.streams[symbol+STREAMTICKER] = stream
-			beWs.mtx.Unlock()
+			beWs.streamLock.Lock()
+			beWs.streams[fmt.Sprintf(STREAM_TICKER_FMT, symbol)] = stream
+			beWs.streamLock.Unlock()
 
-			//Wait for binance to send events
-			time.Sleep(time.Second)
+			if waitForFirstEvent {
+				//Wait for binance to send events
+				time.Sleep(time.Second)
```

Comment:
don't need `waitForFirstEvent` as a bool, it should be a time.Duration = time.Second.
then we can do this: `time.Sleep(waitForFirstEvent)`.